services/api/src/routes/products.js

```diff
@@ -79,6 +91,7 @@ router
   .delete('/:productId', async (ctx) => {
     const product = ctx.state.product;
     await product.delete();
+    await Audit.delete(ctx, 'delete product', product.id);
```

Comment:
I think it might be better to have the action name in the past tense - it makes it easier to use in the admin UI:

- X deleted product Y

Reply:
Yep for sure, note that this is just an example how it could be hooked up very little effort went into adding the action text.
As i mentioned above before this PR is done i will remove it audit util from this endpoint.